src/build-a-bull/src/features/vote-creation/review/index.tsx

```diff
@@ -73,7 +95,7 @@ export default function Review() {
                   threshold: Number(proposal.threshold),
                   ask: Number(proposal.ask),
                 },
-                answers: ['yes'],
+                answers: ['Yes', 'No'],
```

Comment:
what's the purpose of adding `No`?

Reply:
The original `dapp` had a requirement for a minimum of 2 options. Will test without it once we are staged